container/compliance/scan_codecs.py

```
@@ -0,0 +1,240 @@
+                verdict
+            ].append(entry)
+    return violations, exceptions, allowed
+
+
+def _version_lt(a: str, b: str) -> bool:
+    """a < b, best-effort. Uses packaging if available, else a numeric tuple."""
+    try:
+        from packaging.version import Version
+
+        return Version(a) < Version(b)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)